spec/javascripts/social_networking/new_shareable_form_spec.js

```diff
@@ -82,38 +82,7 @@ describe('addSharingToNewShareableForm', function() {
         });
 
         it('formats hidden inputs', function() {
-          expect(getPage().find('[value=planned]').length)
-            .toEqual(1);
-        });
-      });
-
-      describe('on page ready', function() {
```

Comment:
They are being tested above - one was for `page:change` and one was for `ready`.  I updated the one test that references hidden inputs to include a hidden field `[type="hidden"]`so that is why it looks a bit different.